src/index.tsx

```diff
@@ -26,5 +30,25 @@ export function ReactAccordion(props: ReactAccordionProps) {
   );
 }
 
+export function SingleItemOpenAccordion(props: { items: Item[] }) {
+  const { items } = props;
+  const [openItemIndex, setOpenItemIndex] = useState<number | null>(null);
+
+  const handleClick = (event: SyntheticEvent, index:number) => {
+    event.preventDefault();
+    setOpenItemIndex(openItemIndex === index ? null : index);
+  };
+
+  const revisedItems = items.map((item, index) => ({
+    ...item,
+    open: index === openItemIndex,
+  }));
+
+  return (
+    <ReactAccordion items={revisedItems} onClick={handleClick} />
+  );
+}
+
 export default ReactAccordion;
 module.exports = ReactAccordion;
+module.exports.SingleItemOpenAccordion = SingleItemOpenAccordion;
```

Comment:
Is this the best way to export these?

Reply:
Almost definitely not, but I'm not sure of an alternative/better way to handle commonjs support.